tests/unit/ApiService.spec.ts

```diff
@@ -100,4 +100,14 @@ describe('function: createPeginTx', () => {
     expect(result.outputs[2].op_return_data).to.be.equal('test1');
   });
 
+  it('obtain api version, return promise', async () => {
+    setEnvironment(true, true);
+    try {
+      const result = await ApiService.getApiInformation();
+      expect(result.version).not.to.be.null;
+    } catch (e)
+    {
+      expect(e).to.be.a('error', 'Network Error');
```

Comment:
I think an exception should not be catched... I think this is hiding issues.

Reply:
We´re catching only the Network Error, any other errors will broke the tests. It is necessary because we´re testing API endpoint, ie, an external service and that service can be unavailable, causing an Network Error. Only that error will be expected.